src/components/ui/PlaceholderText.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const PlaceholderText = ({ length = 10, tagName = 'div', className }) => {
+  const DomTag = tagName;
+  return <DomTag className={className}>
```

Comment:
I assume this is used to make a container that is a div by default, but that can be modified by changing the tagName param.
Is DomTag necessary? Why not directly &lt;tagName className={className}>?

Reply:
It's related to the way JSX is transpiled to vanilla-JS. JSX needs dynamic component names to start with a capital letter, otherwise it's considered as the html element itself.
So if you write `<tagName />` in JSX, it will render the DOM node `<tagName>`, and we don't want that :)

Reply:
That's very unintuitive :D 
So... just to be sure, it would work if we had:
```
const PlaceholderText = ({ length = 10, TagName = 'div', className }) => {
  return <TagName className={className}>
```
?